front/lib/assistant.ts

```diff
@@ -113,7 +113,7 @@ export function assistantUsageMessage({
   isError,
   shortVersion,
 }: {
-  assistantName: string;
+  assistantName: string | null;
```

Comment:
Why accept null ? This message should not show for assistants not yet created, and for those created we have the name. Refusing null is guarding us from using the message when we shouldn't no?

Reply:
The goal is to match the design where we don't want to repeat the assistant name in the assistant builder and use "This assistant" instead